src/Entity/Conference.php

```diff
@@ -76,7 +76,7 @@ class Conference
     private $startAt;
 
     /**
-     * @ORM\Column(name="end_at", type="datetime")
+     * @ORM\Column(name="end_at", type="datetime", nullable=true)
```

Comment:
🤔 a conference must have an end date, even if it's the same as the start date (one day conference)

Reply:
Some conferences do not have an end date (2 for conf-tech source  for example).
If so, do I put the end date = start date?

Reply:
Ok. No, leave `null` as value